src/components/common/layout.js

```diff
@@ -23,11 +23,19 @@ const globalStyles = css`
     --yellow: #ffba08;
     --denim: #424b71;
     --error: #b5261d;
+    --separator: #efefef;
 
     --aqua: #7dcfb6;
     --gray: #cfd3da;
     --copperfield: #df886c;
 
+    --red-gradient-1: #c5532f;
+    --red-gradient-2: #db7656;
+    --red-gradient-3: #ff9308;
+    --red-gradient-4: var(--yellow);
```

Comment:
a little disconcerting to read `red: yellow;` but I expect it makes sense somewhere

Reply:
yeah I suppose this isn't exactly a red gradient as much as a gradient between red and grey (or whatever that color is), so yellow happens to be on that spectrum. Happy to change the name if you have suggestions